contracts/TokenHolder.sol

```diff
@@ -362,13 +370,22 @@ contract TokenHolder is MultiSigWallet {
         );
 
         // solium-disable-next-line security/no-low-level-calls
-        executeStatus_ = _to.call(_data);
+        executionStatus_ = _to.call(_data);
 
         BrandedToken(brandedToken).approve(tokenRules, 0);
 
         TokenRules(tokenRules).disallowTransfers();
 
-        emit RuleExecuted(messageHash, _nonce, executeStatus_);
+        bytes4 functionSelector = bytesToBytes4(_data);
```

Comment:
We can put a require for EXECUTE_RULE_CALLPREFIX and functionSelector.

Reply:
I am not sure I follow. Here, the functionSelector is not EXECUTE_RULE_CALLPREFIX, indeed it's a rule's function selector that should be called.

Reply:
Sorry my bad. You are correct.